ui/components/app/detected-token/detected-token-ignored-popover/detected-token-ignored-popover.js

```diff
@@ -41,7 +41,10 @@ const DetectedTokenIgnoredPopover = ({
       <Typography
         variant={TYPOGRAPHY.H6}
         tag={TYPOGRAPHY.H6}
-        margin={[0, 5, 7, 5]}
+        marginTop={0}
+        marginRight={5}
+        marginBottom={7}
+        marginLeft={5}
       >
         {t('ignoreTokenWarning')}
       </Typography>
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props